### DISCUSSION TITLE
Help tab not working?

### DISCUSSION THREAD

**Discussion by delta-psi** [posit-dev/positron#3738] [Q&A]
Unsure if anyone else is having this issue with the release on 2024.06.1-27, but for me the 'Help' pane on the right side is not working. I can click on the 'Home' and "Search' icons, but it just flashes the border, and I am unable to use the 'Help' tab like in RStudio. Am I missing something? 

UPDATE: Never mind, just had to run 'help.start()' in the console

**Comment by petetronic** [comment]
Thanks for checking out Positron! The help panel needs some more work on the initial experience and especially the Home screen. We have an [issue](https://github.com/posit-dev/positron/issues/1785) tracking the latter.

**Reply by renatovargas** [reply]
Thank you for working on it. I can confirm that at the time, typing something like `?as.data.frame` in the console will not bring up the help page for that function. Not unless you do `help.start()` first, and this inconsistently too. It sometimes does it, sometimes doesn't. It just flashes the border as explained by the OP. These little glitches aside, Positron is coming together really well. Although it's early stage development, I am amazed at how solid it is already. Thank you.

**Comment by juliasilge** [comment]
I believe these problems are related to https://github.com/posit-dev/positron/issues/3753

**Comment by juliasilge** [comment]
For folks who have been having trouble with the R Help proxy, the [latest release](https://github.com/posit-dev/positron/releases/tag/2024.09.0-1) incorporates the changes we hope will help or solve the problem. Please give it a try!

> [!NOTE]  
> If you as a user continue to see problems (or see different kinds of problems), what is helpful is to [change the R kernel log level to trace](https://github.com/posit-dev/positron/issues/3753#issuecomment-2229528367) and [share the logs from the R Console](https://github.com/posit-dev/positron/issues/3753#issuecomment-2218086901) attached as a file to an issue. Let's have any users with further problems open a new issue or discussion. Thanks!
